nursery/list-containers.yml

```diff
@@ -0,0 +1,18 @@
+rule:
+  meta:
+    name: list containers
+    namespace: host-interaction/container/docker
+    author: william.ballenthin@fireeye.com
+    scope: function
+    att&ck:
+      - Discovery::Container and Resource Discovery [T1609]
+    references:
+      - https://docs.docker.com/engine/api/v1.24/
+    examples:
+  features:
+    - or:
+      - string: /^docker(\.exe)? ps/
+      - and:
+        - match: send HTTP request
+        - string: /\/v1\.[0-9]{1,2}\/containers\/json
```

Comment:
`WARN: rule string quotes incorrect: add double quotes to "/\/v1\.[0-9]{1,2}\/containers\/json"`

Ending `/` is missing.

Reply:
ok linter finally passed! thanks for your patience...